sonar-slang-plugin/src/test/java/com/sonarsource/slang/kotlin/KotlinConverterTest.java

```diff
@@ -123,6 +123,14 @@ public void testFunctionDeclaration() {
     assertTree(emptyLambdaFunction.returnType()).isNull();
     assertThat(emptyLambdaFunction.formalParameters()).isEmpty();
     assertTree(emptyLambdaFunction.body()).isNull();
+
+    FunctionDeclarationTree extensionFunction = (FunctionDeclarationTree) kotlin("fun MutableList.swap(index1: Int, index2: Int) {}");
+    assertTree(extensionFunction.name()).isIdentifier("swap");
```

Comment:
I don't think that we want that. It seems that the `MutableList` part is lost in the AST, which can be a big problem, at least for syntactic equivalence.